pipelines.adoc

```diff
@@ -54,15 +54,9 @@ Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
 
 === Getting started
 
-==== Set up dynamic volume provisioning or a persistent volume
+==== Setting up of a persistent volume for running pipelines
```

Comment:
**Change**
Setting up of a persistent volume for running pipelines
**to**
Set up  a persistent volume to run pipelines

Reply:
looks good.